montecarlo/cfast_mcarlo.py

```diff
@@ -660,7 +662,11 @@ def find_peak_hrr():
         hrr_peak, flashover = find_peak_hrr()
 
         fire_load_d = self.conf['fire_load'][self._comp_type] # [MJ/m2]
-        load_density = int(lognormal(fire_load_d['mean'], fire_load_d['sd']))  # location, scale
+        if fire_load_d['mean']!='' and fire_load_d['sd']!='':
+            load_density = int(lognormal(fire_load_d['mean'], fire_load_d['sd']))  # location, scale
+        elif fire_load_d['1st']!='' and fire_load_d['99th']!='':
```

Comment:
Then error should be raised, since scenario is not set up properly. I'll add relevant lines handling such a situations any time soon.
Nevertheless, this issue should be addressed at the form.php, I suppose.

Reply:
resolved in https://github.com/aamks/aamks/pull/175/commits/d5dab201bf403a0c753b9b656aa5528741a62dc2